homeworks/01-bubble-sort/src/App.tsx

```diff
@@ -1,26 +1,103 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+import { BarSet } from './components/BarSet';
+
+import { createRandomNumberArray } from './utils/appUtils';
+
+const MAX_NUMBER = 100;
+const NUMBER_COUNT = 40;
+const SORT_TIMEOUT = 75;
+
+interface AppProps {}
+
+interface AppState {
+  sorting: boolean,
+  sortingState: {
+    iteration: number,
+    currentIndex: number,
+  },
+  numbers: number[],
+  solved: boolean,
+}
+
+class App extends React.Component<{}, AppState> {
```

Comment:
You have `AppProps `, but don't use it here.

Reply:
Fixed